pygmt/base_plotting.py

```diff
@@ -546,6 +548,7 @@ def plot(self, x=None, y=None, data=None, sizes=None, direction=None, **kwargs):
             quoted lines).
         {W}
         {U}
+        {XY}
```

Comment:
> > 1. Should we have separate `{X}` and `{Y}` entries, or a single `{XY}` entry?
> 
> A single `{XY}` entry looks better.

Ok done. I'll add this to the other functions that support xshift/yshift in a bit.